### DISCUSSION TITLE
Student Developer pack access issue

### DISCUSSION THREAD

**Discussion by yashleo** [community/community#62411] [GitHub Education]
I am unable to get student developer pack access. I had already submitted by College ID car but still I am not getting the access. I am trying this from several months and i have tried it more than 5-10 times but this is not getting further. It is continously shwing that provide an email issued by school. But i dont have it because they dont provde that so what can i do. And now also the continue button this page : "https://education.github.com/discount_requests/application" is not working please help in the case.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @yashleo, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by nonvenomous** [comment]
Same issue since a month for me without any solution or response back from support.
Big thread over here https://github.com/orgs/community/discussions/59360

**Comment by mdxabu** [comment]
> I am unable to get student developer pack access. I had already submitted by College ID car but still I am not getting the access. I am trying this from several months and i have tried it more than 5-10 times but this is not getting further. It is continously shwing that provide an email issued by school. But i dont have it because they dont provde that so what can i do. And now also the continue button this page : "https://education.github.com/discount_requests/application" is not working please help in the case.

you can't get access to the github students developer pack without college/school email ID because the college/school mail ID is mandatory for this program

**Comment by Awrsha** [comment]
Here are some suggestions for troubleshooting your GitHub Student Developer Pack application:

Double check that you are using an email address associated with your school account. Even if your school does not issue emails, GitHub requires a .edu email to verify student status.

If your school truly does not provide emails, contact GitHub Support directly to explain the situation. Provide any official documentation from your school to support this. GitHub may be able to make an exception, but you'll likely need to work with their support team directly.

Make sure you are uploading a high-quality photo or scan of your student ID that includes:

Your full name
The name of your school
A clearly visible issue or expiry date
If the "Continue" button is not working, try clearing your browser cache and cookies, and then reapplying. This sometimes helps resolve issues like non-functional buttons.

As a last resort, ask a teacher, administrator or parent to contact GitHub Support on your behalf. They may have more success getting GitHub to manually review and approve your application due to their official role.

The key is to clearly explain your situation to GitHub Support - that your school does not provide emails, you have uploaded a legible student ID, and the application form is not working correctly. With any luck, they'll be able to verify your student status through other means and approve your application.
